generatorcore/industry2018/i18.py

```diff
@@ -1,13 +1,11 @@
 # pyright: strict
 from dataclasses import dataclass
 from .co2e import CO2e
-from .empty import Empty
 from .co2e_per_t import CO2e_per_t
 from .co2e_with_pct_energy import CO2e_with_pct_energy
 from .co2e_per_mwh import CO2e_per_MWh
 from .co2e_basic import CO2e_basic
-from .energy_pct import Energy_pct
-from .energy_sum import EnergySum
+from .energy_sum import EnergySum, Energy_pct, Empty
```

Comment:
I think Empty and the corresponding member variables should just die. That requires running update_expectations once, but makes the code much nicer.

Reply:
@Ultramarin95, @bgrundmann: Das würde bedeuten, dass s_renew_hydrogen und s_renew_emethan aus I18 rausfliegen. Ich dachte aus Konsistenzgründen, sollten in allen Sektroren alle Energiearten aufgeführt werden, auch wenn sie hier leer sind. Oder wolltest du nur den Typ ändern und Empty rausschmeißen?

Reply:
Also wenn du dir die Sektoren ansiehst die ich schon gemacht habe, da habe ich leere Records auch immer rausgeschmissen.  Also ja s_renew_hydrogen un s_renew_emethan würden verschwinden. Ich meine wir berechnen nix dafür also raus. Macht den Code übersichtlicher und klarer.

Reply:
Done